mailgun/route_handlers.py

```diff
@@ -64,11 +84,7 @@ def handle_mailing_list_email_route(request):
                                u"for %s, but we've already handled that.  "
                                u'Skipping.', recipient, message_id)
                 continue
-
-        try:
-            _handle_recipient(request, recipient)
-        except JsonResponse as error_response:
```

Comment:
This could be a more specific error, with more helpful error content, that you're throwing away by not catching it.